ci/test.sh

```diff
@@ -26,6 +26,31 @@ GTEST_ARGS="--gtest_output=xml:${CUGRAPH_ROOT}/test-results/"
 DOWNLOAD_MODE=""
 EXITCODE=0
 
+PRHTTP=https://api.github.com/repos/rapidsai/cugraph/pulls/${PR_ID}/files
+fnames=`curl -sb -X GET -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $GHTK"  $PRHTTP | python3 -c "import sys, json; print([labels['filename'] for labels in json.load(sys.stdin)])"`
+cpp_changed="false" python_changed="false" nb_changed="false" doc_changed="false" 
+for fname in ${fnames[@]}
+do
+   if [[ "$fname" == *"cpp/cmake/"* || "$fname" == *"cpp/CMakeLists.txt"* || "$fname" == *"cpp/src/"* || "$fname" == *"cpp/include/"* || "$fname" == *"cpp/tests/"* || "$fname" == *"cpp/libcugraph_etl/"* || "$fname" == *"cpp/scripts/"* ]]; then
+      cpp_changed="true" python_changed="true" nb_changed="true" doc_changed="true"
+   fi
+   if [[ "$fname" == *"python/"* ]]; then
+      python_changed="true" nb_changed="true" doc_changed="true"
```

Comment:
It might be better to make the variable names a little more accurate, since `python_changed` implies python code changed when it could actually be set by way of a C++ code change.  Perhaps the naming convention could be `run_python_tests`, `run_nb_tests`, etc.?

Reply:
Make sense